aws/lambda/python/log-anonymiser/module.tf

```diff
@@ -0,0 +1,123 @@
+resource "aws_iam_role" "lambda_s3" {
+  name = "lambda_s3_role"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "s3_logs_lambda" {
+  name = "lambda_s3_policy"
+  role = "${aws_iam_role.lambda_s3.id}"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::cloudfront-logs-openregister",
+        "arn:aws:s3:::cloudfront-logs-register-gov-uk",
+        "arn:aws:s3:::cloudfront-logs-openregister/*",
+        "arn:aws:s3:::cloudfront-logs-register-gov-uk/*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::cloudfront-logs-openregister-anonymised",
```

Comment:
The reason I didn't do this was because the source bucket isn't terraformed and we decided not to terraform that. Since the source bucket would also have to be created manually if we set everything up from scratch, I drew the line between s3 and lambda so only everything in lambda is terraformed.

I'd also prefer for both buckets to be managed the same way as the settings should be consistent across both them.